webpack-modules/README.md

```diff
@@ -0,0 +1,31 @@
+# WebExtension Webpack Example
+A minimal example of how to use npm modules from within a WebExtension.
+The example package used by this extension is `left-pad`, an essential package
+in almost any situation.
+
+## What it does
+This setup takes a background and a content script entrypoint and outputs them
+to their respective target.
```

Comment:
I'm not at all familiar with webpack, so this might just be me, but it took me a little while to understand this. Can we be more explicit, like:

"This example shows how you can use webpack to to package npm modules for a WebExtension. The WebExtension's files are all stored under the "dist" directory. The WebExtension includes a browser action with a popup: the popup provides a UI for left-padding a string with a chosen character.

In this example, a file "popup/left-pad.js" implements the logic behind the popup's UI. This file uses the [left-pad](https://www.npmjs.com/package/left-pad) npm module. We can then use webpack to package this file as a self-contained JS file, which gets written to dist/popup/index.js, where it is included by the WebExtension's popup document."